<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewerWidgetUI</class>
 <widget class="QWidget" name="ViewerWidgetUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1221</width>
    <height>890</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Transform</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget_ControlView" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="btn_PreviousView">
        <property name="icon">
         <iconset resource="contexteditor.qrc">
          <normaloff>:/REMainWindow/icons/light/back.svg</normaloff>:/REMainWindow/icons/light/back.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>12</width>
          <height>12</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btn_NextView">
        <property name="icon">
         <iconset resource="contexteditor.qrc">
          <normaloff>:/REMainWindow/icons/light/next.svg</normaloff>:/REMainWindow/icons/light/next.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>12</width>
          <height>12</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btn_CloseView">
        <property name="icon">
         <iconset resource="contexteditor.qrc">
          <normaloff>:/REMainWindow/icons/light/cross.svg</normaloff>:/REMainWindow/icons/light/cross.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>12</width>
          <height>12</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="combo_view">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>22</height>
         </size>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_Spacer" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_MouseCoords" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_PosX">
             <property name="text">
              <string>X=</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_ValuePosX">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>?</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_PosY">
             <property name="text">
              <string>Y=</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_ValuePosY">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>?</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_PosZ">
             <property name="text">
              <string>Z=</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_ValuePosZ">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>?</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QStackedWidget" name="view_stack">
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="contexteditor.qrc"/>
 </resources>
 <connections/>
</ui>
